build_msvc/README.md

```diff
@@ -12,7 +12,7 @@ Quick Start
 The minimal steps required to build Bitcoin Core with the msbuild toolchain are below. More detailed instructions are contained in the following sections.
 
 ```
-vcpkg install --triplet x64-windows-static boost-filesystem boost-signals2 boost-test libevent openssl zeromq berkeleydb rapidcheck double-conversion
+vcpkg install --triplet x64-windows-static boost-filesystem boost-multi-index boost-signals2 boost-test boost-thread libevent openssl zeromq berkeleydb rapidcheck double-conversion
```

Comment:
Why are they required here, but not in https://github.com/bitcoin/bitcoin/blob/08e29473126d5cc4df6d2b3f368c6f6f641c0bd8/.appveyor.yml#L10 ?

Reply:
I'd assume our appveyor image is using an older version of `vcpkg`.

Reply:
Yep, looks like `boost-multi-index` used to be built as part of `boost-signals2`, but that [recently changed](https://github.com/microsoft/vcpkg/commit/9433136b22b5e443d550ca7665a562637c8a86b9#diff-4bdbb65fe1fa280aae9a260f6139daab). Assume the same is true for `boost-thread`.